cmd/feedback/feedback.go

```diff
@@ -377,6 +377,7 @@ func chooseSurveyPrompt(ctx context.Context, clients *shared.ClientFactory, surv
 	selection, err := clients.IO.SelectPrompt(ctx, msg, surveyPromptOptions,
 		iostreams.SelectPromptConfig{
 			Flag:     clients.Config.Flags.Lookup("name"),
+			Required: true,
```

Comment:
🔍 The `required` option defaulting to **false** will cause this prompt to skip from the changes below!